crates/pathfinder/src/ethereum/contract.rs

```diff
@@ -44,9 +44,7 @@ pub fn addresses(chain: Chain) -> ContractAddresses {
         Chain::Mainnet => MAINNET_ADDRESSES,
         Chain::Goerli => GOERLI_ADDRESSES,
         #[cfg(test)]
-        // https://external.integration.starknet.io/feeder_gateway/get_contract_addresses
-        // {"Starknet": "0xd5c325D183C592C94998000C5e0EED9e6655c020", "GpsStatementVerifier": "0xAB43bA48c9edF4C2C4bB01237348D1D7B28ef168"}
-        Chain::Integration => todo!("Add support for integration chain."),
+        Chain::Integration => GOERLI_ADDRESSES,
```

Comment:
This is incorrect. Only the mempage address is shared. The core and gps are the ones returned by the `get_contract_addresses` call. So you will need to add an `INTEGRATION_ADDRESSES`.